lib/csv_to_list_converter.dart

```diff
@@ -235,11 +233,11 @@ class CsvToListSink extends ChunkedConversionSink<String> {
 
         if (result.stopReason == ParsingStopReason.EndOfString) {
           if (_currentRow.isNotEmpty && end) {
-            _outSink.add(_currentRow);
+            _outSink.add([_currentRow]);
```

Comment:
I think that the chunked conversion idea, with these correct types, supports that you could build up a "table" (list of rows) and push them all at once, for one chunk, to outSink. Instead of always adding single-element lists of one row. But I'll leave that for later.

Reply:
The types are unfortunately no longer correct.

LineSplitter had the same problem.

I can not accept your merge request without incrementing the version.  Fixing the types is on my ToDo list (since quite some time :(

If I can't find time in the next weeks to fix the types completely I will simply increment version and apply your pull request.